HW2/04-background.js

```diff
@@ -3,7 +3,7 @@ let IntTime = 3;
 
 
 function start_stop(){
```

Comment:
Just two suggestions here: 
- on line 27: "return "rgb(" + r + "," + g + "," + b + ")";" Caterina suggests using string interpolation so line 27 would turn into return `rgb(${r},${g},${b})`; (git is removeing my backtics here but keep in mind this is wrapped in backtics.
- The assignment suggested using rgba colors looks like rgb is being used this would just change line 27 to be "rgba(R,G,B,A)" where A stands for an alpha decimal between 0 and 1.

Other than that I like how you delegated out responsibility for changing colors, creating colors, and handling the start and stop of the background change it keeps it easy to follow and organized. This file looks great you have consistent double quote use for strings, and you use the correct logical comparisons.